_data/filter-projects.yml

```diff
@@ -0,0 +1,9 @@
+- title: All Projects
+  url: /projects/
+  category: allProjects
+- title: Member
+  url: /projects/member/
```

Comment:
We've called these Organization projects in the charter and in the proposed badges.

Reply:
oh oops, good catch